lock-api/src/main/java/com/palantir/lock/client/TransactionStarter.java

```diff
@@ -45,38 +44,32 @@
  * A service responsible for coalescing multiple start transaction calls into a single start transactions call. This
  * service also handles creating {@link LockTokenShare}'s to enable multiple transactions sharing a single immutable
  * timestamp.
- *
+ * <p>
  * Callers of this class should use {@link #unlock(Set)} and {@link #refreshLockLeases(Set)} for returned lock tokens,
  * rather than directly calling delegate lock service.
  */
 final class TransactionStarter implements AutoCloseable {
-    private final DisruptorAutobatcher<Void, StartIdentifiedAtlasDbTransactionResponse> autobatcher;
+    private final DisruptorAutobatcher<Integer, List<StartIdentifiedAtlasDbTransactionResponse>> autobatcher;
```

Comment:
This is a bit sucky, since we're having to allocate Integers, while disruptor is optimised for that and reuses the actual instances of arguments, but I don't think it's worth obsessing about this. @jeremyk-91